MethodsHomework/MethodsHomework.cs

```diff
@@ -109,52 +109,52 @@ public void UsingMethods()
 
             // Find a method to reverse this list of ages.
             // hint, uses the dot operatore on ages. like ages.SomeMethod()
-            
+            ages.Reverse();
             Grading.ListIsReversed(ages);
 
             // find a method to add a value to the list ages at the end.
             int addMeIntoList = 12;
-            
+            ages.Add(addMeIntoList);
             Grading.TwelveWasAdded(ages);
 
             // find a method to add a value to the list at the front of the list ages, ie, the start of it, or first value.
             int addMeToTheListAtTheVeryFront = 1;
-            
+            ages.Insert(0, addMeToTheListAtTheVeryFront);
             Grading.OneWasAddedToTheFront(ages);
 
             // find a method to clear the list ages.
-            
+            ages.Clear();
             Grading.AgesIsNowEmpty(ages);
 
             // find a method to check if the double amount is negative.
 
             double amount = 29.3;
             // you must capture the result of the found method;
-            bool isNegative = /*Answer here*/;
+            bool isNegative = double.IsNegative(amount);/*Answer here*/
             Grading.AmountIsNotNegative(isNegative);
 
             // Find a method to tell you if the sentence contains the work fox
             string sentence = "The quick brown fox.";
             // you must capture the result of the found method;
-            bool doesContaintFox = /*Answer here*/;
+            bool doesContaintFox = sentence.Contains("fox");/*Answer here*/
             Grading.ContainsFox(doesContaintFox);
 
             // Find a method to replace the word quick, with the word slow
             // you must capture the result of the found method;
-            sentence.Replace("quick", "slow");
-            string newSentence = /*Answer here*/;
+            
+            string newSentence = sentence.Replace("quick", "slow");/*Answer here*/
             Grading.ReplacedTheWord(newSentence);
 
             // Find a method to Convert this word to lowercase;
             string word = "UPPER";
             // you must capture the result of the found method;
-            string lowered = /*Answer here*/;
+            string lowered = word.ToLower();/*Answer here*/
             Grading.StringWasLowercased(lowered);
 
             // Find a method to Convert this word to lowercase;
             string needsSplit = "I,Need,These,Separate";
             // you must capture the result of the found method;
-            string[] split = /*Answer here*/;
+            string[] split = needsSplit.Split(',',4);/*Answer here*/
```

Comment:
This works.
Just so you're aware, though, `Split()` has another overload that doesn't need a length. Most of the time, that's the overload you want. 

```C#
string needsSplit = "This,is,a,long,string,with,many,words";

string[] splitFour = needsSplit.Split(',', 4);
// splitFour is equivalent to:
// new string[] { "This", "is", "a", "long,string,with,many,words" }

string[] splitUnlimited = needsSplit.Split(',');  // Note: No second argument
// splitUnlimited is equivalent to:
// new string[] { "This", "is", "a", "long", "string", "with", "many", "words" }
```